### PR TITLE
Adagio: Update adagio.json

### PR DESCRIPTION
**Pull request by aboncimino**
removed second colon from prebidVersion parameter

### CODE REVIEW

File: static/bidder-params/adagio.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
         "^[a-zA-Z_]": { "type": "string" }
       }
     },
-    "prebidVersion:": {
```
Comment (mansinahar) [inline static/bidder-params/adagio.json:53]: Can you please add tests for this and other parameters supported by your adapter in `adagio/params_test.json`. Currently, it only covers `organizationId`, `site` and, `placement` parameters? Adding tests for these other parameters will help catch these errors sooner.

**Comment by mansinahar** [comment]
@aboncimino Friendly reminder about the comment above

**Comment by osazos** [comment]
Hi @mansinahar, 
@aboncimino is not par of the Adagio company. I'll fix the error and tests asap.

**Comment by SyntaxNode** [comment]
> Hi @mansinahar,
> @aboncimino is not par of the Adagio company. I'll fix the error and tests asap.

Gotcha. Shall we close this PR in favor of a PR to be opened by the Adagio team?

**Comment by osazos** [comment]
Yes please close it. I'm fixing it.

**Comment by SyntaxNode** [comment]
@aboncimino Thank you identifying the issue. As noted in this thread, the Adagio team would like to fix this themselves and will do asap.

**Comment by hhhjort** [comment]
@osazos : any ETA for the fix?

**Pull request closed without merging**